src/main/java/appeng/services/version/ModVersionFetcher.java

```diff
@@ -30,11 +53,18 @@ public Version get()
 		{
 			return new DoNotCheckVersion();
 		}
-		else
+
+		try
 		{
 			final Version version = this.parser.parse( this.rawModVersion );
 
 			return version;
 		}
+		catch( final Exception e )
```

Comment:
Nearly no user will ever encounter an `AssertionError` as like nobody starts their client with `-ea`, thus unit tests are more or less the only place they are thrown.

In the related issues an `IllegalArgumentException` bubbles up, due to the disabled assertions.

If the intention for the assertions is that they should always be triggered, then `Preconditions.checkArgument()` is probably a better alternative.